src/main/haskell/language-kore/src/Data/Kore/AST.hs

```diff
@@ -578,8 +582,8 @@ data SentenceAxiom = SentenceAxiom
 from the Semantics of K, Section 9.1.6 (Declaration and Definitions).
 -}
 data SentenceSort = SentenceSort
-    { sentenceSortParameters :: ![UnifiedSortVariable]
-    , sentenceSortSort       :: !(Sort Object)
+    { sentenceSortName       :: !(Id Object)
+    , sentenceSortParameters :: ![UnifiedSortVariable]
```

Comment:
Maybe `![Sort Object]` for `sentenceSortParameters` too?

Reply:
The PDF says "<sort-declaration> ::= ‘sort’ <object-sort-constructor> ‘{’ <sort-variable-list> ‘}’ <attribute>", so that's what I did.